dev/dependencyList

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 
+HikariCP/4.0.3//HikariCP-4.0.3.jar
```

Comment:
Add deps should update LICENSE and NOTICE.

Reply:
updated the license, seems notice is not needed, the new added two dependencies are all apache lincense